src/GitHub.App/Extensions/AkavacheExtensions.cs

```diff
@@ -34,7 +38,7 @@ public static class AkavacheExtensions
         {
             return Observable.Defer(() =>
             {
-                var absoluteExpiration = blobCache.Scheduler.Now + maxCacheDuration;
+                var absoluteExpiration = blobCache.Scheduler.Now.ToUniversalTime() + maxCacheDuration;
```

Comment:
So I poked further, and it's not that Scheduler.Now returns a `DateTimeOffset`, it's that the  implementation actually returns `SystemClock.UtcNow`, which is why there's no difference between the two. Because the property is called `Now` and not `UtcNow`, I thought it returned `DateTimeOffset.Now`, which is local time and definitely not what we want. Sigh at API inconsistencies...

As long as that doesn't change, it's all good.